### ISSUE TITLE
Drowsiness Detection using EEG signals 

### ISSUE THREAD

**Issue by KamakshiOjha** [Niketkumardheeryan/ML-CaPsule#809]
Aim : To develop a deep learning model to detect drowsiness from EEG signals using various algorithms and compare their performance to identify the best-fitted algorithm based on accuracy scores.

 Approach :

1. Exploratory Data Analysis (EDA):
- Analyze and preprocess the EEG dataset.
- Visualize EEG signals and class distributions.

2. Model Development:
- Implement the provided deep learning model using TensorFlow and Keras.
- Develop and implement three additional models: Convolutional Neural Network (CNN).

3. Model Training and Evaluation:
- Train each model on the EEG dataset.

4. Visualization and Conclusion:
- Visualize training and validation metrics.

Please assign me this issue under GSSOC.

**Comment by KamakshiOjha** [comment]
Please assign me this issue under GSSOC.

**Comment by KamakshiOjha** [comment]
Submitted the Pull request for PR no - #809 Drowsiness Detection using EEG signals #782, and a previous Pull request for PR no - #772 --> Face Mask Detection using Machine learning(DL), Both are submitted
